.ansible-lint

```diff
@@ -7,3 +7,5 @@ offline: false
 use_default_rules: true
 parseable: true
 verbosity: 1
+warn_list:
+  - '106'
```

Comment:
algum motivo em especial pra desabilitar esse warning?

Reply:
Esse warning está ocorrendo devido ao nome da role colocado na mentoria
[E106] Role name iac-role-node-rke does not match ``^[a-z][a-z0-9_]+$`` pattern
Então como faz parte do projeto decidimos deixar na parte de ignore do lint

Reply:
Vamos criar uma issue para renomear os repositórios de ansible com undescore (_) ao inves de hífen (-)

Reply:
https://github.com/mentoriaiac/iac-role-node-rke/issues/3

Reply:
Boa! Vamos deixar isso como débito técnico.